Documentation/GettingStartedContributing.md

```diff
@@ -0,0 +1,62 @@
+# Getting started contributing to Ladybird
+Welcome to the Ladybird web browser project!
+
+This document aims to be a beginner-friendly summary of the existing contribution requirements for Ladybird, remember to read the linked documentation for more information.
+
+## Getting familiar with the project
+Ladybird is a large project making use of many homegrown and third-party libraries and programs, written primarily in C++.
+
+It is recommend you read the FAQs in case it already answers any questions you have, there is also a [Discord server](https://discord.gg/nvfjVJ4Svh), which is the preferred way to ask the maintainers and community for help.
```

Comment:
```suggestion
It is recommend you read the FAQ in case it already answers any questions you have; there is also a [Discord server](https://discord.gg/nvfjVJ4Svh), which is the preferred way to ask the maintainers and community for help.
```